### PR TITLE
Update Team CodeBender's team page for end of week 4

### PR DESCRIPTION
**Pull request by thngkaiyuan**

### CODE REVIEW

File: teams/codebender.md [hunk 1]
```diff
@@ -9,15 +9,51 @@ permalink: /teams/codebender/
  - Thng Kai Yuan ([@thngkaiyuan](http://cs3281-2.blogspot.sg/))
    - Technology Expertise: Information Security
  - Moon Seokmin ([@bsmmoon](http://moonmoon3281.blogspot.sg/))
-   - Technology Expertise:
+   - Technology Expertise: Web Crawling
```
Comment (lowjoel) [inline teams/codebender.md:12]: @bsmmoon  in what sense? as a search engine, or?

File: teams/codebender.md [hunk 2]
```diff
@@ -9,15 +9,51 @@ permalink: /teams/codebender/
...
+* [ ] Create test case to ensure show info page shows correct info - Week 9
+* [ ] Create dashboard page - Week 9
+* [ ] Create test case to ensure dashboard page shows correct and links correctly - Week 10
+* [ ] Create log/analytics database - Week 10
+* [ ] Create Analytics page - Week 11
+* [ ] Create test case to ensure Analytic page shows correct analytics - Week 11
+* [ ] Create admin page - Week 12
+* [ ] Create test case for admin page and features - Week 12
+
+## Workplan for Eratosthenis
```
Comment (lowjoel) [inline teams/codebender.md]: Eratosthenes

Comment (thngkaiyuan) [inline teams/codebender.md]: Thanks for spotting that typo, fixed in 049e90084a263c8cc41c27ead446af7269a318ab.

File: teams/codebender.md (hunk 1 above)
Comment (lowjoel) [inline teams/codebender.md]: @bsmmoon

Comment (bsmmoon) [inline teams/codebender.md:12]: @lowjoel  Mostly web scraping. But I am also interested in other usage such as internet bot as well.

Comment (lowjoel) [inline teams/codebender.md:12]: yes, the question is, how can it be phrased as a skill, like something that other people do not have.

Comment (damithc) [inline teams/codebender.md:12]: SEO?

Comment (bsmmoon) [inline teams/codebender.md:12]: What about Information Retrieval?

**Pull request merged**